src/index.js

```diff
@@ -1,2 +1,40 @@
-// Import styles
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/225524368-888d9fc3-0be7-49de-b115-4b62d15972a3.png)

Since you don't have any database so it would be great if you remove this Database `details` tag to make your readme file clean and professional 😊